shim/src/main.rs

```diff
@@ -446,6 +447,12 @@ fn get_version() -> &'static str {
 }
 
 fn main() -> Result<()> {
+    let mut updater =
+        // check for updates
+        UpdateNotifier::new(String::from("turbo"), Some(String::from("latest")), None);
```

Comment:
```suggestion
        UpdateNotifier::new("turbo".to_string(), Some("latest".to_string()), None);
```

`to_string` (or `to_owned`) is generally the ideomatic way of doing this (afaik)

Reply:
Nice, thanks!